Variance_Correlation_Plot.R

```diff
@@ -0,0 +1,557 @@
+transformed.communication.activity["Subject"] <- (apply(transformed.communication.activity, 1, ParseSubject, 2))
+
+# Subject HCR001 
+HCR001.4months <- subset(transformed.communication.activity, Subject == "HCR001")
+HCR001.4months["Subject"] <- NULL
+HCR001.4months["TimeSubjectIndex"] <- NULL
+
+postscript("HCR001_Variance.4months.eps",  width = 480, height = 480)
+PlotR2s(HCR001.4months, "Variance Explained in HCR001 (all 4 months)",
+        "ActCom")
+dev.off()
+
+# HCR003
+HCR003.4months <- subset(transformed.communication.activity, Subject == "HCR003")
+HCR003.4months["Subject"] <- NULL
+HCR003.4months["TimeSubjectIndex"] <- NULL
+postscript("HCR003_Variance.4months.eps",  width = 480, height = 480)
+PlotR2s(HCR003.4months, "Variance Explained in HCR003 (all 4 months)",
+        "ActCom")
+dev.off()
+
+# HCR004
+HCR004.4months <- subset(transformed.communication.activity, Subject == "HCR004")
+HCR004.4months["Subject"] <- NULL
+HCR004.4months["TimeSubjectIndex"] <- NULL
+
+postscript("HCR004_Variance.4months.eps",  width = 480, height = 480)
+PlotR2s(HCR004.4months, "Variance Explained in HCR004 (all 4 months)",
+        "ActCom")
+dev.off()
+
+# HCR006
+HCR006.4months <- subset(transformed.communication.activity, Subject == "HCR006")
+HCR006.4months["Subject"] <- NULL
+HCR006.4months["TimeSubjectIndex"] <- NULL
+
+postscript("HCR006_Variance.4months.eps",  width = 480, height = 480)
+PlotR2s(HCR006.4months, "Variance Explained in HCR006 (all 4 months)",
+        "ActCom")
+dev.off()
+
+# HCR008
+HCR008.4months <- subset(transformed.communication.activity, Subject == "HCR008")
+HCR008.4months["Subject"] <- NULL
+HCR008.4months["TimeSubjectIndex"] <- NULL
+postscript("HCR008_Variance.4months.eps",  width = 480, height = 480)
+PlotR2s(HCR008.4months, "Variance Explained in HCR008 (all 4 months)",
+        "ActCom")
+dev.off()
+
+# HCR009
+HCR009.4months <- subset(transformed.communication.activity, Subject == "HCR009")
+HCR009.4months["Subject"] <- NULL
+HCR009.4months["TimeSubjectIndex"] <- NULL
+
+postscript("HCR009_Variance.4months.eps",  width = 480, height = 480)
+PlotR2s(HCR009.4months, "Variance Explained in HCR009 (all 4 months)",
+        "ActCom")
+dev.off()
+
+
+# All four months (no blood pressure/heart rate data)
+transformed.communication.activity["TimeSubjectIndex"] <- NULL
+transformed.communication.activity["Subject"] <- NULL
+postscript("Variability.Activity.Communication.eps",  width = 480, height = 480)
+PlotR2s(transformed.communication.activity,
+        "Variance Explained in Activity and Communication Data", "ActCom")
+dev.off()
+
+
+######################################
+
+# All four months by subject (with energy)
+Energy.4months["Subject"] <- (apply(Energy.4months, 1, ParseSubject, 2))
+Energy.4months["X1"] <- NULL
+Energy.4months["Times"] <- NULL
+# Subject HCR001 
+HCR001.4months <- subset(Energy.4months, Subject == "HCR001")
+HCR001.4months["Subject"] <- NULL
```

Comment:
I'm wondering why "Subject" and "TimeSubjectIndex" are getting dropped for every individual here? Is there a reason this shouldn't be handled by logic in `PlotR2s` _or_ be dropped before you plot for individuals?